lib/screen/login/login_page.dart

```diff
@@ -136,19 +146,3 @@ class _LoginPageState extends State<LoginPage> {
     ]);
   }
 }
-
-class CloseButton extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Positioned(
```

Comment:
Flutter was complaining about this Positioned guy cause it was inside a Stack.

No idea wtf that means but when I got rid of it it went away. It also looks the same when I tested it.

Reply:
@bstrumillo